db/migrate/20240122202020_relax_notifications_schema_for_sms.rb

```diff
@@ -0,0 +1,15 @@
+class RelaxNotificationsSchemaForSms < ActiveRecord::Migration[7.1]
```

Comment:
I agree with the approach you used here. We never finished implementing the enum stuff for notifications anyhow, and this makes it a lot more flexible.